doc/classes/GraphEdit.xml

```diff
@@ -181,6 +181,14 @@
 				[/codeblocks]
 			</description>
 		</method>
+		<method name="get_connection_count">
+			<return type="int" />
+			<param index="0" name="from_node" type="StringName" />
+			<param index="1" name="from_port" type="int" />
+			<description>
+				Returns the number of connections from [param from_node] to [param from_port] port.
```

Comment:
```suggestion
				Returns the number of connections from [param from_port] of [param from_node].
```

Reply:
> 

Since I've had a lot of trouble with commit suggestions in the past, to commit and not create a new commit, should I click Commit suggestion or add suggestion to batch ? 😅

Reply:
Adding to batch will just add one commit for all suggestions. You have to manually copy the suggestion if you want to avoid GitHub's commits.